types/debug/index.d.ts

```diff
@@ -28,6 +30,7 @@ declare namespace debug {
 
     export interface IDebugger {
         (formatter: any, ...args: any[]): void;
+        (...args: string[]): void;
```

Comment:
It's also pointless to declare this overload - you might as well have written it as `(): void` since the original overload permits 1 or more arguments.